api/src/test/java/org/openmrs/module/cohort/api/dao/CohortGenericDaoTest.java

```diff
@@ -147,4 +149,11 @@ public void getByUuid_shouldReturnNullForVoidedOrRetiredCohort() {
 		
 		assertThat(dao.get(COHORT_UUID), nullValue());
 	}
+	
+	@Test
+	public void shouldVoidCohortM() {
+		CohortM cohortToVoid = dao.get(COHORT_UUID);
+		Context.getService(CohortService.class).voidCohortM(cohortToVoid, "delete cohort");
```

Comment:
After calling service.voidCohort, can you fetch the cohort again from the dao and then assert that it is voided?